integrationtests/cdi-jcache-it/src/test/java/org/infinispan/integrationtests/cdijcache/interceptor/CacheRemoveAllInterceptorTest.java

```diff
@@ -49,6 +53,16 @@ public void beforeMethod() {
       assertTrue(customCache.isEmpty());
    }
 
+   @BeforeTest(alwaysRun = true)
+   protected void testClassStarted(ITestContext context) {
```

Comment:
Should use `@Listeners(TestResourceTrackingListener.class)` instead, it's less noisy.